### DISCUSSION TITLE
The host **** does not support SSL connections.

### DISCUSSION THREAD

**Discussion by shuangyuxiaoyi** [quartznet/quartznet#652] [General] [answered]
I want to use mysql to log, instead of RAMJobStore.
Mysql version is 8.0.
And give this error when execute the sql commond that picture
![111](https://user-images.githubusercontent.com/26379840/43313540-e70ca398-91c2-11e8-8c4f-9a8011625897.PNG)

**Comment by shuangyuxiaoyi** [comment]
additional the my scheduler  constructor picture is
![image](https://user-images.githubusercontent.com/26379840/43313632-33be4c32-91c3-11e8-91fd-bd8527b7496e.png)

**Comment by lahma** [comment, marked as answer]
You need to define that you don't want SSL in your connection string:

https://stackoverflow.com/questions/45086283/mysql-data-mysqlclient-mysqlexception-the-host-localhost-does-not-support-ssl/45108611

You should also use MySQLDelegate instead of plain StdAdoDelegate (in your configuration), it performs better.
